pos_warning_exiting/static/src/js/chrome.js

```diff
@@ -0,0 +1,87 @@
+odoo.define("pos_warning_exiting.Chrome", function (require) {
+    "use strict";
+
+    const {_t} = require("web.core");
+    const Chrome = require("point_of_sale.Chrome");
+    const Registries = require("point_of_sale.Registries");
+    const {isRpcError} = require("point_of_sale.utils");
+
+    var _super_chromemodel = Chrome.prototype;
+
+    const PosWarningChrome = (Chrome) =>
+        class extends Chrome {
+            // @Override
+            async _closePos() {
+                var self = this;
+                const draft_orders_description = [];
+                const order_jsons = this.env.pos.db.get_unpaid_orders();
+
+                if (order_jsons.length) {
+                    order_jsons.forEach(function (order_json) {
+                        if (
+                            order_json.pos_session_id === self.env.pos.pos_session.id &&
+                            order_json.lines.length
+                        ) {
+                            var order_description =
+                                _t("Order #") + order_json.sequence_number;
+                            // Lazy depedency to pos_restaurant
+                            // to display table information
+                            if (order_json.table) {
+                                order_description +=
+                                    _t(" - Table: ") + order_json.table.name;
+                            }
+                            draft_orders_description.push(order_description);
+                        }
+                    });
+                }
+
+                if (draft_orders_description.length) {
+                    const reason = this.env._t(
+                        "You have some draft unpaid orders." +
+                            " You can exit temporarily the Point of Sale, but you" +
+                            " will loose that orders if you close the session: " +
+                            draft_orders_description.join(", ")
+                    );
```

Comment:
Blocking.

This won't ever be translated because you're including `draft_orders_description` into the translatable string.

You need to do something like this:
```js
const message = this.env._t(
    "You have some draft unpaid orders. "
    + "You can exit temporarily the Point of Sale, but you "
    + "will loose that orders if you close the session: "
);
const reason = message + draft_orders_description.join(", ");
```